plenum/test/view_change_with_delays/helper.py

```diff
@@ -223,7 +223,7 @@ def do_view_change_with_propagate_primary_on_one_delayed_node(
     view_no = lpc[0]
 
     with delay_rules(slow_stasher, icDelay()):
```

Comment:
I think it would be great to make sure that data is equal at the end of `do_view_change_with_propagate_primary_on_one_delayed_node` and `do_view_change_with_delay_on_one_node`

Reply:
Also we need to check that `waiting_for_new_view` is False on all nodes